src/tiledimage.js

```diff
@@ -1499,11 +1499,14 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
 
         // Load the new 'best' n tiles
         if (bestTiles && bestTiles.length > 0) {
-            for (let tile of bestTiles) {
-                if (tile) {
-                    this._loadTile(tile, currentTime);
+            // Avoid executing this in the frame loop
+            setTimeout(() => {
```

Comment:
This makes for better framerates as requesting data through ajax is costly operation so avoid doing it on the animation frame.

Reply:
Nice 😊